src/mngt/ocf_mngt_cache.c

```diff
@@ -2391,7 +2402,8 @@ ocf_mngt_cache_stop_standby_pipeline_properties = {
 	.finish = ocf_mngt_cache_stop_finish,
 	.steps = {
 		OCF_PL_STEP(ocf_mngt_cache_stop_wait_metadata_io),
-		OCF_PL_STEP(ocf_mngt_cache_stop_deinit_metadata),
+		OCF_PL_STEP(ocf_mngt_cache_deinit_metadata),
+		OCF_PL_STEP(ocf_mngt_cache_deinit_cache_volume),
```

Comment:
I think you meant to reverse the order here as well

Reply:
Fixed.